### DISCUSSION TITLE
Pairing Problem

### DISCUSSION THREAD

**Discussion by GLoren1** [Navid200/xDrip#156] [Dexcom]
My G6 transmitter got disconnected from Xdrip. I 've been attempting to pair the device again, unsuccessfully. The event log indicates that I need to manually pair the device. I have tried doing that through my Android. When I attempt to pair through the Android bluetooth setting, I am able to click on DexcomG6, but it never pairs.  Any help would be appreciated.

**Comment by Navid200** [comment]
You should never attempt to pair from the Android Bluetooth settings page.

The only way you should pair is by approving the xDrip pair request, which pops up on screen.

Please post a screenshot of your system status page.

**Comment by GLoren1** [comment]
![temp](https://user-images.githubusercontent.com/62320462/145857612-3e968092-4652-4253-9d0b-c6ebb2ce3273.png)

**Reply by Navid200** [reply]
Is there a reason engineering mode is enabled?

**Reply by GLoren1** [reply]
Not sure other than I had to hard reset the transmitter a few months ago.  I don't know whether that turns on engineering mode.

FYI, at the moment, the transmitter is paired to Dexcom app on another phone that I began using after I couldn't pair with xDrip.

**Reply by Navid200** [reply]
No it doesn't and you cannot hard reset this transmitter anyway.
Please disable engineering mode.

Are all settings correct?
https://github.com/Navid200/xDrip/wiki/G6-Recommended-Settings

**Comment by GLoren1** [comment]
Yes. They are correct.

**Reply by Navid200** [reply]
You can temporarily change the transmitter serial number, under Hardware Data Source, and change it back to the correct value again.
That will trigger a pair request.   It may take up to 5 minutes.
You need to approve that request.

**Reply by GLoren1** [reply]
That worked. Thank you.

**Reply by GLoren1** [reply]
II may have spoke too soon.  My connection disconnected again, and I repeated the steps of temporarily changing the transmitter serial number and changing it back.  And, it still won't reconnect.

**Reply by Navid200** [reply]
You cannot use both Dexcom app and xDrip as collectors from the same transmitter at the same time.

**Comment by GLoren1** [comment]
After I turned off the Dexcom app, this problem was solved.  I have another issue though that I'm about to post.
